markdown.go

```diff
@@ -179,7 +180,7 @@ type Renderer interface {
 
 	// Span-level callbacks
 	AutoLink(out *bytes.Buffer, link []byte, kind int)
-	CodeSpan(out *bytes.Buffer, text []byte)
+	CodeSpan(out *bytes.Buffer, text []byte, lang string)
```

Comment:
I'll point out that this changes the public `Renderer` interface, that means it will make every existing implementation in the wild no longer import [`blackfriday.Renderer` interface](https://godoc.org/github.com/russross/blackfriday#Renderer). This doesn't mean it cannot be done, but it should not be taken lightly and discussed before doing.